### PR TITLE
HEEDLS-528 Refactor Tracking system views to fix inconsistent spacing

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
More spacing fixes, this time for:
Edit centre manager details:
image

Edit centre website details:
image

Delegate approvals reject page:
image

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreManagerDetails.cshtml [hunk 1]
```diff
@@ -12,30 +12,57 @@
...
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
     @if (errorHasOccurred) {
-      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Telephone) })"/>
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Telephone) })" />
     }
 
-    <h1 id="page-heading" class="nhsuk-heading-xl">Edit centre manager details</h1>
```
Comment (DanBloxham-sw) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreManagerDetails.cshtml:24]: Is this id for the page heading not used for anything?

**Review by DanBloxham-sw — APPROVED** [review]
Looks good. Question about the page-heading id, but doesn't need rereview.

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreManagerDetails.cshtml (hunk 1 above)
Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreManagerDetails.cshtml:24]: As far as I can tell, outside of the LearningMenu, no other than messing with the styles by reducing the bottom margin

**Review by stellake — APPROVED** [review]
Looks good! 👍

**Pull request merged**